.gitignore

```diff
@@ -5,6 +5,7 @@
 
 #Ignore byond config folder.
 /cfg/**/*
+/config/admins.txt
```

Comment:
ahhhhh

Reply:
This file shouldn't be committed, it is individual for whoever is hosting a test server. The one in the repo works for you but doesn't work for everyone else.